symfony/plugins/orangehrmAdminPlugin/modules/integration/templates/_osIntegration.php

```diff
@@ -39,10 +40,52 @@
     });
 
     <?php echo htmlspecialchars_decode($page['js']); ?>
+    
+    function togglSyc() {
```

Comment:
Why OS need to know something about toggle plugin? it should be part of toggle add-on

Reply:
renamed as ajaxSyc